frontend/public/extend/devconsole/components/topology/topology-utils.ts

```diff
@@ -106,7 +105,7 @@ export class TransformTopologyData {
   private _selectorsByService;
   private _allServices;
 
-  constructor(public resources: TopologyDataResources) {
+  constructor(public resources: TopologyDataResources, public application: string) {
```

Comment:
```suggestion
  constructor(public resources: TopologyDataResources, public application?: string) {
```
This way you can omit the application argument instead of having to pass in `undefined`.

Reply:
fixed in https://github.com/talamer/console/pull/110/commits/aecd608a6dd0c102c8904ef9f00733a7876cd05e